_config.yml

```diff
@@ -517,93 +468,79 @@ t:
-        logo: "https://i.postimg.cc/NfW4Xyx7/camera-logo.png"
-        content: "El nostre soci català per a l'adaptació del nostre dispositiu a les càmeres existents"
-        height: 80
-      -
-        title: "FORCE 06"
-        url: "https://www.departement06.fr/preservation-du-cadre-de-vie/force-06-3080.html"
-        logo: "/img/force06_logo.png"
-        content: "El nostre soci al departament dels Alpes Marítims per diversificar les proves de camp"
-        height: 80
-      -
-        title: "SDIS 12"
-        url: "https://www.sdis12.fr/"
-        logo: "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSg2bUbkvsTyiTM7uHqbHBY0nxzu7L9AGHEGg&s"
-        content: "El nostre soci a Aveyron per diversificar les proves de camp"
-        height: 125
-      -
-        title: "SDIS 77"
-        url: "https://www.sdis77.fr/"
-        logo: "https://www.infirmiersapeurpompier.com/wp-content/uploads/2020/11/77.jpg"
-        content: "El nostre soci a Seine&Marne per diversificar les proves de camp"
-        height: 75
+        - title: "SDIS 07"
+          url: "https://www.sdis07.fr/"
+          logo: "https://www.sdis07.fr/ardeche-secours-incendie/themes/sapeurs-pompiers-ardeche/medias/img/logo-sdis07-pompiers-ardeche.png"
+          content: "El nostre primer soci per a la recollida de requisits i l'experimentació de camp"
+          height: 75
+        - title: "Engie Green"
+          url: "https://www.engie-green.fr/"
+          logo: "https://www.engie-green.fr/wp-content/themes/engie-green/assets/img/engie-green-logo.svg"
+          content: "El nostre primer soci privat, amb qui estem fent un experiment a Gironda"
+          height: 80
+        - title: "CENIA"
+          url: "https://cenia.cl/"
+          logo: "/img/cenia.jpg"
+          content: "El Centre Nacional d'Intel·ligència Artificial de Xile amb qui estem treballant per al nostre primer desplegament a l'estranger"
+          height: 60
+        - title: "ADF Subirats"
+          url: "https://subirats.cat/adfsubirats/"
+          logo: "https://i.postimg.cc/NfW4Xyx7/camera-logo.png"
+          content: "El nostre soci català per a l'adaptació del nostre dispositiu a les càmeres existents"
+          height: 80
+        - title: "FORCE 06"
+          url: "https://www.departement06.fr/preservation-du-cadre-de-vie/force-06-3080.html"
+          logo: "/img/force06_logo.png"
+          content: "El nostre soci al departament dels Alpes Marítims per diversificar les proves de camp"
+          height: 80
+        - title: "SDIS 12"
+          url: "https://www.sdis12.fr/"
+          logo: "https://encrypted-tbn0.gstatic.com/images?q=tbn:ANd9GcSg2bUbkvsTyiTM7uHqbHBY0nxzu7L9AGHEGg&s"
+          content: "El nostre soci a Aveyron per diversificar les proves de camp"
+          height: 125
+        - title: "SDIS 77"
+          url: "https://www.sdis77.fr/"
+          logo: "https://www.infirmiersapeurpompier.com/wp-content/uploads/2020/11/77.jpg"
```

Comment:
The new logo is missing in catalan translation, could you point to the new one  as in english ? thks